Lib/fontTools/varLib/__init__.py

```diff
@@ -502,6 +503,30 @@ def _add_gvar(font, model, master_ttfs, tolerance=0.5, optimize=True):
 
 			gvar.variations[glyph].append(var)
 
+def _add_cvar(font, model, master_ttfs, tolerance=0.5):
+
+	log.info("Generating cvar")
+	assert "cvar" not in font
+	cvar = font["cvar"] = newTable('cvar')
+	cvar.version = 1
+	cvar.variations = []
+
+	allCVTs = [CVTValues(m["cvt "].values) for m in master_ttfs]
+	num_cvts0 = len(allCVTs[0])
+
+	if (any(len(c) != num_cvts0 for c in allCVTs)):
+		log.warning("Master has incompatible cvt table, not building cvar table.")
+		del font["cvar"]
+		return
+	deltas = model.getDeltas(allCVTs)
+	supports = model.supports
+	for i,(delta,support) in enumerate(zip(deltas[1:], supports[1:])):
+		delta = [int(round(d)) for d in delta]
```

Comment:
just `round` here is fine, don't need to cast to int as we are using the python3's round which always return an int when used with one argument